README.md

```diff
@@ -1,2 +1,28 @@
-# github robot
-github robot
+# ci-bot
+[![Go Report Card](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)
+[![Build Status](https://travis-ci.org/Huawei-PaaS/ci-bot.svg?branch=master)](https://travis-ci.org/Huawei-PaaS/ci-bot)
+[![LICENSE](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://github.com/Huawei-PaaS/ci-bot/blob/master/LICENSE)
+
+This repository houses ci bot which aims to provide more simple and rich features for the github project users.
+
+## Build
+
+Before you are getting started, you must have [Go](https://golang.org/) already installed in your local machine.
```

Comment:
Before you are getting started, you must have -> Before you get started, make sure to have

Reply:
I will. thanks